.github/workflows/ci.yml

```diff
@@ -112,9 +112,9 @@ jobs:
           fetch-depth: 0
 
       - name: Setup node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
-          node-version: '14'
```

Comment:
I think we can bump these versions in `migration.yml` as well

Reply:
#811